dev/tests/Unit/Command/ComponentInfoCommandTest.php

```diff
@@ -46,9 +45,7 @@ class ComponentInfoCommandTest extends TestCase
 
     public static function setUpBeforeClass(): void
     {
-        $application = new Application();
-        $application->add(new ComponentInfoCommand());
-        self::$commandTester = new CommandTester($application->get('component-info'));
+        self::$commandTester = new CommandTester(new ComponentInfoCommand());
```

Comment:
This is just cleaning up / simplifying these tests, as we can just create the `CommandTester` directly when we don't need to test things like Question Helpers (e.g. `$this->getHelper('question')`)